tests/integration/api/test_event_view_operations.py

```diff
@@ -1370,18 +1370,13 @@ def test_non_float_tile_value_added_to_tile_table(event_view, source_type):
     """
     Test case to ensure non-float tile value can be added to an existing tile table without issues
     """
-    # Make the aggregation_id unique in this test case. Otherwise, the tile cache would have been
-    # calculated using the tile id version 2, causing a tile table not found error. This is a test
-    # specific issue due to the patch and doesn't happen in actual workflow.
```

Comment:
This was wrong. This is actually a scenario that needs to be handled. Without the changes in this PR this test would fail.